### PR TITLE
Add Catalonian Language

### PR DESCRIPTION
**Pull request by M4rtik4**
## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: new translation
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [X] I have followed the [contribution guidelines](https://github.com/UI-Lovelace-Minimalist/UI/blob/main/.github/CONTRIBUTING.md)
    - [ ] This PR is for a custom-card or documentation change and therefore directed to the `main` branch.
    - [x] This PR is for a official card or any other directly to the integration related change and therefore directed to the `release` branch.

### CODE REVIEW

**Review by Borty97 — APPROVED** [review]

File: custom_components/ui_lovelace_minimalist/translations/ca.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (Borty97) [inline custom_components/ui_lovelace_minimalist/translations/ca.json:1]: Correct translation. Good work!

**Review by Borty97 — CHANGES_REQUESTED** [review]

**Comment by basbruss** [comment]
New languages should also be added here to be picked up by the config flow.
https://github.com/UI-Lovelace-Minimalist/UI/blob/f9724d256130d1150469e41fe6a8db022705edf2/custom_components/ui_lovelace_minimalist/const.py#L21C1-L60C2

File: custom_components/ui_lovelace_minimalist/translations/ca.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                    "community_cards": "Selecciona les Tarjetes de comunitat que vols activar."
+                }
+            }
+        },
+        "abort": {
+            "single_instance_allowed": "Només es permet una única configuració de UI Lovelace Minimalist.",
+            "not_setup": "UI Lovelace Minimalist no està preparat."
+        },
+        "error": {
+            "github_cards": "Alguna cosa ha anat malament al obtenir tarjetes de GitHub.",
```
Comment (Borty97) [inline custom_components/ui_lovelace_minimalist/translations/ca.json:55]: Change "tarjetes" for "targetes"

Comment (M4rtik4) [reply custom_components/ui_lovelace_minimalist/translations/ca.json]: Done

**Review by basbruss — APPROVED** [review]

**Pull request merged**